### PR TITLE
docs: adjust readme according to suggestions

### PR DESCRIPTION
**Pull request by JoanFM**
Goals:

Adjust to suggestions to help people follow readme in an easier way

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,9 +195,13 @@ Or you can simply pull it from Jina Cloud:
 left_da = DocumentArray.pull('demo-leftda', show_progress=True)
 ```
 
+**Note**
+If you have at least XGB of RAM and want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.
```
Comment (NicholasDunham) [inline README.md:199]: Should "XGB" be a specific number of gigabytes of RAM? "At least XGB" doesn't make sense to me.

File: README.md [hunk 2]
```diff
@@ -195,9 +195,13 @@ Or you can simply pull it from Jina Cloud:
...
+
+
 You will see a running progress bar to indicate the downloading process.
 
-To get a feeling of the data you will handle, plot them in one sprite image:
+To get a feeling of the data you will handle, plot them in one sprite image, you will need to have matplotlib and torch installed to run this snippet:
```
Comment (NicholasDunham) [inline README.md:204]:
```suggestion
To get a feeling of the data you will handle, plot them in one sprite image. You will need to have matplotlib and torch installed to run this snippet:
```

**Comment by BenjaminEliaMueller** [comment]
With the X amount of RAM I meant an approximate minimum of usable RAM that you need to have, in order to execute the readme example with the full dataset. I cannot test the actual RAM usage of a run with the full dataset myself since I have only 16G available.

When trying out the example at todays Hackathon (at Jina in Berlin) everyone who tested the examples was running into this issue.

**Comment by JoanFM** [comment]
> With the X amount of RAM I meant an approximate minimum of usable RAM that you need to have, in order to execute the readme example with the full dataset. I cannot test the actual RAM usage of a run with the full dataset myself since I have only 16G available.
> 
> When trying out the example at todays Hackathon (at Jina in Berlin) everyone who tested the examples was running into this issue.

yes X amount is a placeholder, I have to compute this. I have tried now and I could not manage to have it working without crashing and I have 16 GB RAM.

How much memory u need to run this?

```python

from docarray import DocumentArray

left_da = DocumentArray.pull('demo-leftda', show_progress=False)

from docarray import Document


def preproc(d: Document):
    return (
        d.load_uri_to_image_tensor()  # load
            .set_image_tensor_normalization()  # normalize color
            .set_image_tensor_channel_axis(-1, 0)
    )  # switch color axis for the PyTorch model later


left_da.apply(preproc)

import torchvision

model = torchvision.models.resnet50(pretrained=True)  # load ResNet50
left_da.embed(model, device='cpu')

right_da = (
    DocumentArray.pull('demo-rightda', show_progress=True)
        .apply(preproc)
        .embed(model, device='cpu')
)

left_da.match(right_da, limit=9)
```

File: README.md (hunk 1 above)
Comment (JoanFM) [reply README.md:199]: yes, I am trying to figure X out

File: README.md [hunk 3]
```diff
@@ -195,9 +195,13 @@ Or you can simply pull it from Jina Cloud:
 left_da = DocumentArray.pull('demo-leftda', show_progress=True)
 ```
 
+**Note**
+You will need more than 15GB of RAM if you want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.
```
Comment (NicholasDunham) [inline README.md:199]:
```suggestion
If you have more than 15GB of RAM and want to try using the whole dataset instead of just the first 1000 images, remove [:1000] when loading the files into the DocumentArrays left_da and right_da.
```

**Review by NicholasDunham — CHANGES_REQUESTED** [review]
One more wording change. Other than that, looks good.

**Review by NicholasDunham — APPROVED** [review]
LGTM
